frame/nomination-pools/src/migration.rs

```diff
@@ -52,9 +52,12 @@ pub mod v1 {
 
 	impl<T: Config> OldBondedPoolInner<T> {
 		fn migrate_to_v1(self) -> BondedPoolInner<T> {
+			// TODO: I don't think this is correct, commission shouldn't be included in
+			// this migration.
 			BondedPoolInner {
-				member_counter: self.member_counter,
 				points: self.points,
+				commission: Commission::default(),
```

Comment:
@kianenigma I added `commission` here to keep the compiler happy with `BondedPoolInner`, but I am assuming `commission` shouldn't be present here as it did not exist on v0 - v1.

Would changing this to another type like `OldBondedPoolInner` be the right thing to do here?

Reply:
Added a note to communicate `commission` field was added in mod4